### ISSUE TITLE
Best Practice: Benchmarks

### ISSUE THREAD

**Issue by expede** [fission-suite/web-api#254]
As a general best practice, having benches set up helps to understand which parts of the application are worth investing time and energy into optimizing for machine execution rather than product execution 😉

**Comment by JonathanLorimer** [comment]
I would be interested in trying my hand at this. Which sub-directory would be worth tackling first? I assumed `fission-core` but may be wrong.

**Comment by expede** [comment]
@JonathanLorimer yeah, literally anything would be helpful! `-core` is the mostly primitives. There's also already a handful of microbenchmarks in there, so possibly a good place to start! Cheers :)
